content/show/the-launch/0014.md

```diff
@@ -0,0 +1,195 @@
+      },
+      {
+        "startTime": 1450,
+        "title": "Boosts",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 1822,
+        "title": "Squatter Included in Price",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 1975,
+        "title": "Slight Wind Damage",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 2250,
+        "title": "Outro",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      }
+    ],
+    "author": null,
+    "title": "14: Game On!",
+    "podcastName": "The Launch 🚀",
+    "description": null,
+    "fileName": null,
+    "waypoints": null
+  },
+  "podcast_alt_file": null,
+  "podcast_ogg_file": null,
+  "video_file": null,
+  "video_hd_file": null,
+  "video_mobile_file": null,
+  "youtube_link": null,
+  "jb_url": "https://www.jupiterbroadcasting.com/show/the-launch/14",
+  "fireside_url": "https://www.weeklylaunch.rocks/episodepage/14",
+  "value": {
+    "type": "lightning",
+    "method": "keysend",
+    "suggested": 5e-8,
+    "recipients": [
+      {
+        "name": "Chris",
+        "type": "node",
+        "address": "03de23d27775ff1abc1d5770e56ee058464c9fcd4cc39837e605646e95aaf5f8f4",
+        "customKey": null,
+        "customValue": null,
+        "split": 30,
+        "fee": false
+      },
+      {
+        "name": "JB Node",
+        "type": "node",
+        "address": "037d284d2d7e6cec7623adbe600450a73b42fb90800989f05a862464b05408df39",
+        "customKey": null,
+        "customValue": null,
+        "split": 20,
+        "fee": false
+      },
+      {
+        "name": "Angerz",
+        "type": "node",
+        "address": "022be9317c82371b76a8e57f96139fc9537f0cee52c649488405072c8c7ab4d601",
+        "customKey": "696969",
+        "customValue": "14",
+        "split": 30,
+        "fee": false
+      },
+      {
+        "name": "Editor Drew",
+        "type": "node",
+        "address": "03df3b998533a072aaf6296195610264738fbf0d9f85d6a7473ece9a80d31ea478",
+        "customKey": null,
+        "customValue": null,
+        "split": 10,
+        "fee": false
+      }
+    ],
+    "timesplits": [
+      {
+        "startTime": 1300,
+        "duration": 162,
+        "remotePercentage": 95,
+        "remoteStartTime": null,
+        "remoteItem": {
+          "feedGuid": "e3c75491-ccb2-5add-82c8-78a5d1dff7a0",
+          "feedUrl": "https://wavlake.com/feed/3cc3847a-8d5f-4086-9501-c1500165da07",
+          "itemGuid": "8f0a54e4-9835-4ec1-a9ef-fe65a8c11db9",
+          "medium": "music"
+        },
+        "recipients": []
+      }
+    ]
+  }
+}
+
+
+### Episode Links
+
+****
```

Comment:
I'm not gonna stop the episode from publishing but I assume something was missing in the feed description?

Reply:
It's the phone number thrown in